src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -55,6 +73,74 @@ def _update_log_filenames(self):
                     )
                 )
 
+    def _load_structured_log_record(self, record: str):
+        try:
+            data = ast.literal_eval(record)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I'm assuming you've used `ast.literal_eval` because the logs may not be parsed by the JSON module.
Can you add a comment for this and also add a note on `ast.literal_eval`'s safety? For example, "literal_eval only accepts a subset of Python code and cannot be used to perform malicious actions"

Reply:
Good catch, forgot to comment that. 

Parsing logs via `.literal_eval()` couples our logs format with Python syntax. We shouldn't do that unless there's a very good reason to.